app/assets/stylesheets/govuk_publishing_components/components/_input.scss

```diff
@@ -16,3 +16,24 @@
     padding-left: govuk-spacing(6);
   }
 }
+
+.gem-c-input__wrapper {
+  display: flex;
```

Comment:
Do we need a fallback for this? I'm guessing probably not but thought it was worth asking.

Reply:
Excellent point, Andy. I've updated the code to fallback to display inline for IE9 and below and used the prefixed property for IE10. Also added screen captures to the PR description to ease the review.